### PR TITLE
Remove unused components and views

### PR DESCRIPTION
**Pull request by csc-felipe**
### Description

Since migrating to the new UI, some components and views have become unused, and this PR removes them.

### Mentions

Used https://www.npmjs.com/package/vue-unused-components-checker to look for unused components.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -95,6 +95,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix hiding the pagination of data tables
 - Fix shared objects functionality: visibility, deleting, editing tags
 
+### Removed
+
+- Unused components and views
```
Comment (blankdots) [inline CHANGELOG.md:100]: would be good to name the views, as it is easier to track

**Review by hannyle — DISMISSED** [review]
LGTM! 👍

File: CHANGELOG.md (hunk 1 above)
Comment (csc-felipe) [reply CHANGELOG.md]: updated

**Comment by csc-felipe** [comment]
Added removed views to the changelog

**Review by hannyle — APPROVED** [review]

**Review by blankdots — APPROVED** [review]

**Pull request merged**